docs/content/02-tutorials/06-downloading.md

```diff
@@ -0,0 +1,60 @@
+
++++
+title = "d. Downloading ⇓"
+weight = 25
++++
+
+This tutorial shows you how you can download a file from an external source at cluster start.
+
+## Setup
+
+In order to download a file when you start your cluster,
+
+1. From the Pcluster Manager GUI create a new cluster. Select the **Wizard** option:
+2. On the **HeadNode** configuration tab
+    + Expand **Advanced Options** > turn on **Multi-Script Runner** > select **Downloader**
+
+![Cloud9 Screenshot](cloud9/cloud9-2.png)
+
+*Note*: You can specify either an `http://`, `https://` or `s3://` endpoint, however for any s3 location it must reside in the same region as the cluster.
+
+*Note*: You may specify any number of additional arguments to the script and it will download each of them to the destination directory.
+
+
+3. On the final screen review and make sure your config looks similar to the following. The only required parameter is the `cloud9.sh` script, we also use the `SSMManagedInstanceCore` policy to connect to the headnode.
+
+```yaml
+HeadNode:
+  InstanceType: t2.micro
+  Ssh:
+    KeyName: keypair
+  Networking:
+    SubnetId: subnet-123456789
+  CustomActions:
+    OnNodeConfigured:
+      Script: >-
+        https://raw.githubusercontent.com/aws-samples/pcluster-manager/main/resources/scripts/multi-runner.py
+      Args:
+        - >-
+          https://raw.githubusercontent.com/aws-samples/pcluster-manager/main/resources/scripts/downloader.sh
+        - '-/tmp'
+        - '-https://aws.amazon.com'
+        - '-s3://mybucket/myfile'
```

Comment:
what happens if one of the URL's returns a 403?

Reply:
Errors will prevent the cluster from creation